topics/application-gradle-plugin.md

```diff
@@ -0,0 +1,68 @@
+[//]: # (title: Distribution via Gradle Application plugin)
+
+From the Gradle [documentation](https://docs.gradle.org/current/userguide/application_plugin.html):
```

Comment:
I'd not cite Gradle docs because they can be changed. Also, tips are intended to add some additional info or hints and don't suit for citation.
It's better to add a clear introduction and describe which this help topic is aimed for. It's OK to copy/paste from their docs and write smth like this: `The Gradle [Application plugin] (https://docs.gradle.org/current/userguide/application_plugin.html) facilitates ... . In this topic, we'll show how to ... .`